File: exercises/src/01 CompositeUI/before/Divergent.Frontend/app/modules/sales/orders/ordersController.js
(function () {
    angular.module('app.controllers')
        .controller('ordersController',
        ['$log', 'backendCompositionService',
            function ($log, backendCompositionService) {

                var ctrl = this;

                ctrl.isBusy = null;
                ctrl.orders = null;

                ctrl.isBusy = backendCompositionService
                    .get('orders-list', { pageIndex: 0, pageSize: 10 })
                    .then(function (viewModel) {
                        ctrl.orders = viewModel.orders;
                    });

            }]);
}())

File: demos/asp-net-core/Divergent.Website/Controllers/OrdersController.cs
using Microsoft.AspNetCore.Mvc;

namespace Divergent.Website.Controllers
{
    public class OrdersController : Controller
    {
        [HttpGet("/orders")]
        public IActionResult Index(int? pageIndex, int? pageSize)
        {
            /*
             * Pagination is not used. It's kept in the code
             * to demonsrate:
             * - how to propagatr it down to composition handlers
             * - that it's owned by Sales
             */
            return View();
        }

        [HttpGet("/orders/details/{id}")]
        public IActionResult Details(int id)
        {
            return View();
        }
    }
}